README.md

```diff
@@ -25,8 +50,7 @@ To use this you need to the following steps:
 1. call `` python3 generatePredictions.py``
 1. Upload the ``output.txt`` that was generated to the ImageTagger
 
-FCN (Fully Convolutional Neural Network)
```

Comment:
Is this chapter still relevant?
Where to find this FCN?

Reply:
It is, because the YOLO approach can not properly output tilted bounding boxes.  So I suggest using it if you need more precision than a regular bounding box. The code for it can be found in the FCN folder.